app/components/common/highLightedContent/index.tsx

```diff
@@ -79,60 +59,3 @@ export function getHighlightedContent(content: string, highlightText: string) {
     })
     .join(' ');
 }
-
```

Comment:
This file did not return component. How about change this file name? and change file location too. (Looks closer to helper logic)